CONTRIBUTING.md

```diff
@@ -1,42 +1,42 @@
+* Fix project itself: `php php-cs-fixer fix`.
 
-You can do some things to increase the chance that your pull request is accepted the first time:
+## Opening a [Pull Request](https://help.github.com/articles/about-pull-requests/)
 
- * Submit one pull request per fix or feature.
- * If your changes are not up to date - rebase your branch on the parent branch.
- * Follow the conventions used in the project.
- * Remember about tests and documentation.
- * Don't bump version.
+You can do some things to increase the chance that your Pull Request is accepted the first time:
 
-## Making new fixers
+* Submit one Pull Request per fix or feature.
+* If your changes are not up to date, [rebase](https://git-scm.com/docs/git-rebase) your branch onto the parent branch.
+* Follow the conventions used in the project.
+* Remember about tests and documentation.
+* Don't bump version.
 
-There is a [cookbook](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/COOKBOOK-FIXERS.md) with basic instructions on how to build a new fixer. Consider reading it
+## Making New Fixers
+
+There is a [cookbook](doc/COOKBOOK-FIXERS.md) with basic instructions on how to build a new fixer. Consider reading it
 before opening a PR.
 
-## Project's standards
+## Project's Standards
 
- * [PSR-1: Basic Coding Standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md)
- * [PSR-2: Coding Style Guide](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)
- * [PSR-4: Autoloading Standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-4-autoloader.md)
- * [PSR-5: PHPDoc (draft)](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md)
- * [Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html)
- * [Symfony Documentation Standards](http://symfony.com/doc/current/contributing/documentation/standards.html)
- * Keep the order of class elements: static properties, instance properties, constructor (or setUp for PHPUnit), destructor (or tearDown for PHPUnit), static methods, instance methods, magic static methods, magic instance methods.
+* [PSR-1: Basic Coding Standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md)
+* [PSR-2: Coding Style Guide](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)
+* [PSR-4: Autoloading Standard](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-4-autoloader.md)
+* [PSR-5: PHPDoc (draft)](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md)
+* [Symfony Coding Standards](https://symfony.com/doc/current/contributing/code/standards.html)
+* Keep the order of class elements: static properties, instance properties, constructor (or setUp for PHPUnit), destructor (or tearDown for PHPUnit), static methods, instance methods, magic static methods, magic instance methods.
```

Comment:
it's exception on the list - all the rest are standards that are well documented and provided by some entity (PHP-FIG, Sf), the last point was exception as we wanted to follow it, yet we didn't had fixer for that. Now we have. We could drop this point IMO.